packages/apps/plugins/plugin-function/src/components/TriggerEditor.tsx

```diff
@@ -46,8 +46,8 @@ export const TriggerEditor = ({ space, trigger }: { space: Space; trigger: Funct
   const fn = useMemo(() => query.find((fn) => fn.uri === trigger.function), [trigger.function, query]);
 
   useEffect(() => {
-    void space.db.schemaRegistry
-      .getAll()
+    void space.db.schema
+      .listDynamic()
```

Comment:
i think just `get()` (and if necessary pass in options) to match the other APIs.

Reply:
return types are different, agreed to leave as is